Duplicati/Library/Backend/Backblaze/B2.cs

```diff
@@ -163,6 +181,7 @@ public IList<ICommandLineArgument> SupportedCommands
                     new CommandLineArgument("auth-username", CommandLineArgument.ArgumentType.String, Strings.B2.AuthUsernameDescriptionShort, Strings.B2.AuthUsernameDescriptionLong),
                     new CommandLineArgument(B2_CREATE_BUCKET_TYPE_OPTION, CommandLineArgument.ArgumentType.String, Strings.B2.B2createbuckettypeDescriptionShort, Strings.B2.B2createbuckettypeDescriptionLong, DEFAULT_BUCKET_TYPE),
                     new CommandLineArgument(B2_PAGESIZE_OPTION, CommandLineArgument.ArgumentType.Integer, Strings.B2.B2pagesizeDescriptionShort, Strings.B2.B2pagesizeDescriptionLong, DEFAULT_PAGE_SIZE.ToString()),
+                    new CommandLineArgument(B2_DOWNLOAD_URL_OPTION, CommandLineArgument.ArgumentType.String, Strings.B2.B2downloadurlDescriptionShort, Strings.B2.B2downloadurlDescriptionLong),

```

Comment:
I would recommend using the overloaded `CommandLineArgument` method that allows defining default value.

Then you can delete the part of the long description that states the default value.

`public CommandLineArgument(string name, ArgumentType type, string shortDescription, string longDescription, string defaultValue, string[] aliases)
            : this(name, type, shortDescription, longDescription, defaultValue)`

Reply:
Actually there is no default hardcoded value for `downloadUrl`. I put it in the description as `https://f00X.backblazeb2.com` where `X` depends on the account. Maybe I need to clarify it in the description a little bit. Thanks

Reply:
Oh, that makes more sense